test/classes/initializers/parentCall/inParallel.good

```diff
@@ -0,0 +1 @@
+inParallel.chpl:7: error: super.init() call in parallel statement makes Phase for other statements ambiguous
```

Comment:
I'd probably just say something like "illegal use of super.init() in parallel statement"